Modules/ThirdParty/NIFTI/src/nifti/niftilib/nifti1_io.h

```diff
@@ -109,7 +109,7 @@ typedef enum _analyze75_orient_code {
            nifti1 format definition; it is used to implement one API
            for reading/writing formats in the nifti1 format.
  */
-typedef struct {                /*!< Image storage struct **/
+typedef struct nifti_image_struct {                /*!< Image storage struct **/
```

Comment:
@blowekamp When this is approved, let's make sure to update the upstream to add this named structure.

Reply:
First, I don't think this PR should change niftilib itself at all.  That should be separate.

But anyway, I think better for ITK to just use `struct nifti_image` than to change upstream. It would complicate regression testing different versions of ITK and different versions of niftilib.